projects/ion/src/lib/input/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface IonInputProps {
+  /**
+   * key of the input
```

Comment:
I think this description is better this way
```suggestion
   * Indicates whether the input is invalid.
```